dll/cpl/sysdm/lang/ro-RO.rc

```diff
@@ -34,236 +34,236 @@ END
+    PUSHBUTTON "Se&tări", IDC_STAREC, 194, 162, 50, 15
+    PUSHBUTTON "Se&tări sistem…", IDC_SYSSETTINGS, 6, 192, 80, 15
+    PUSHBUTTON "&Variabile de mediu", IDC_ENVVAR, 88, 192, 80, 15
+    PUSHBUTTON "&Raportare erori", IDC_ERRORREPORT, 170, 192, 80, 15
 END
 
 IDD_SYSSETTINGS DIALOGEX 0, 0, 221, 106
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Opțiuni de sistem"
+CAPTION "Setări de sistem"
 FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
     GROUPBOX "Informații versiune", IDC_STATIC, 6, 3, 210, 73
     LTEXT "ReactOS este construit ca sistem server și se raportează ca atare. Bifați această căsuță pentru a dedica sistemul exclusiv aplicațiilor.", IDC_STATIC, 15, 15, 183, 41
-    CONTROL "Solicit &autoraportarea sistemului ca stație de lucru", IDC_REPORTASWORKSTATION, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 16, 57, 190, 10
-    PUSHBUTTON "Î&nchide", IDOK, 166, 83, 50, 14
+    CONTROL "&Raportare ca stație de lucru", IDC_REPORTASWORKSTATION, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 16, 57, 190, 10
+    PUSHBUTTON "OK", IDOK, 166, 83, 50, 14
 END
 
 IDD_HARDWAREPROFILES DIALOGEX 6, 18, 254, 234
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Profiluri ale dispozitivelor"
+CAPTION "Profiluri hardware"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_HARDPROF, IDC_STATIC, 8, 11, 18, 20, SS_ICON
-    LTEXT "Puteți configura profilurile hardware pentru diferite configurații ale dispozitivelor. La pornire puteți alege profgilul pe care doriți să îl folosiți.", IDC_STATIC, 46, 11, 188, 25
-    LTEXT "Profiluri disponibile:", IDC_STATIC, 8, 41, 120, 8
+    LTEXT "Aveţi posibilitatea să iniţializaţi profiluri hardware pentru diferite configuraţii hardware. La pornire, aveţi posibilitatea să alegeţi profilul dorit.", IDC_STATIC, 46, 11, 188, 25
+    LTEXT "&Profiluri hardware disponibile:", IDC_STATIC, 8, 41, 120, 8
     LISTBOX IDC_HRDPROFLSTBOX, 8, 52, 215, 54, LBS_NOTIFY
     PUSHBUTTON "", IDC_HRDPROFUP, 232, 59, 15, 14, BS_ICON | WS_DISABLED
     PUSHBUTTON "", IDC_HRDPROFDWN, 232, 79, 15, 14, BS_ICON | WS_DISABLED
-    PUSHBUTTON "&Proprietăți…", IDC_HRDPROFPROP, 8, 106, 50, 14, WS_DISABLED
-    PUSHBUTTON "&Copiere…", IDC_HRDPROFCOPY, 63, 106, 50, 14, WS_DISABLED
-    PUSHBUTTON "&Redenumire…", IDC_HRDPROFRENAME, 118, 106, 50, 14, WS_DISABLED
-    PUSHBUTTON "&Eliminare…", IDC_HRDPROFDEL, 173, 106, 50, 14, WS_DISABLED
-    GROUPBOX "Selecție profiluri ale dispozitivelor", IDC_STATIC, 7, 130, 240, 75
-    LTEXT "La pornirea ReactOS:", IDC_STATIC, 14, 142, 210, 8
-    AUTORADIOBUTTON "&Așteaptă alegerea explicită a unui profil", IDC_HRDPROFWAIT, 14, 158, 190, 8, WS_GROUP
-    AUTORADIOBUTTON "Alege a&utomat primul profil în lipsa unei alegeri timp de", IDC_HRDPROFSELECT, 14, 173, 190, 8
+    PUSHBUTTON "Propr&ietăţi", IDC_HRDPROFPROP, 8, 106, 50, 14, WS_DISABLED
```

Comment:
Ms-ro-RO-pic proving the &i accelerator please.

Reply:
the 50 width do look like a very close fit as well. Possible that it overflows already.

Reply:
![2024-05-05 17_59_39-Resource Hacker - sysdm cpl mui](https://github.com/reactos/reactos/assets/113553683/242a2975-14e6-4984-abf0-d2c0b5c784be)

Reply:
Thanks. Resolved.